packages/protocol/contracts/common/LibConstStrings.sol

```diff
@@ -0,0 +1,87 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.24;
+
+/// @title LibConstStrings
+/// @custom:security-contact security@taiko.xyz
+library LibConstStrings {
+    /// @notice Keccak hash of the string "STATE_ROOT".
+    bytes32 internal constant HASH_STATE_ROOT = keccak256("STATE_ROOT");
+
+    /// @notice Keccak hash of the string "SIGNAL_ROOT".
+    bytes32 internal constant HASH_SIGNAL_ROOT = keccak256("SIGNAL_ROOT");
+
+    /// @notice Constant string "SIGNAL".
+    string internal constant SIGNAL = "SIGNAL";
+
+    /// @notice bytes32 representation of the string "chain_pauser".
+    bytes32 internal constant BYTES32_CHAIN_PAUSER = bytes32("chain_pauser");
```

Comment:
@adaki2004 I still suggest that we use shorter leading string, "H_" for hash, "B_" for bytes32, "S_" for string. I even suggest to rename this to "Strings.sol" to make it shorter.

Reply:
Done, but used `LibString` instead, since `Strings` bc. it could have been misleading. (OZ's Strings module)